packages/ckeditor5-find-and-replace/theme/findandreplaceform.css

```diff
@@ -149,4 +149,78 @@
 			}
 		}
 	}
+}
+
+.ck[dir='rtl'] {
+	& .ck .ck-find-and-replace-form .ck-find-form__wrapper .ck-labeled-field-view__input-wrapper .ck.ck-input-text {
+		padding: var(--ck-spacing-extra-tiny) var(--ck-spacing-medium) var(--ck-spacing-extra-tiny) calc(var(--ck-spacing-medium) * 8);
+	}
+
+	& .ck .ck-find-and-replace-form  .ck-replace-form__wrapper {
+		& .ck-replace-buttons {
+			display: flex;
+			flex-flow: row-reverse;
+
+			& .ck-button:last-child {
+				margin-left: var(--ck-spacing-standard)
+			}
+		}
+		& .ck-labeled-field-view__input-wrapper .ck.ck-input-text{
+			padding: var(--ck-spacing-extra-tiny) var(--ck-spacing-medium) var(--ck-spacing-extra-tiny) var(--ck-spacing-medium)
+		}
+	}
+
+	& .ck.ck-responsive-form > :not(:last-child) {
+		margin-left: 0;
+	}
+
+	& .ck .ck-find-and-replace-form .ck-find-form__wrapper {
+		& .ck-button.ck-button-prev,
+		& .ck-button.ck-button-next {
+			margin-right: 1px;
+			margin-left: 0;
+		}
+
+		& span.ck-results-counter {
+			left: calc(var(--ck-spacing-standard) * 3);
+		}
+
+		& .ck-find-buttons {
+			flex-flow: row-reverse;
+
+			& .ck-button svg.ck-button__icon {
```

Comment:
I just scanned this PR and how does this rule differs from this rule? [https://github.com/ckeditor/ckeditor5/blob/i/9981/packages/ckeditor5-find-and-replace/theme/findandreplaceform.css#L93](https://github.com/ckeditor/ckeditor5/blob/i/9981/packages/ckeditor5-find-and-replace/theme/findandreplaceform.css#L93)

The way I see it rule I linked still applies to rtl so this particular rule here is redundant. Your goal should be to repeat as little as possible - that's very important.